ui/src/app/job-list/table/table.component.html

```diff
@@ -67,7 +67,43 @@
     <ng-container matColumnDef="{{ df.field }}">
       <mat-header-cell *matHeaderCellDef>{{ df.display }}</mat-header-cell>
       <mat-cell class="additional-column" *matCellDef="let j">
-        <div class="display-field" *ngIf=!isStatusField(df)>{{ getFieldValue(j, df) }}</div>
+        <div class="display-field" *ngIf=canEdit(df)>
+          <div [ngSwitch]="getFieldType(df)">
+            <div *ngSwitchCase="'text'">
+              <div class="edit-field">
+                <span class="field-value">{{ getFieldValue(j, df) }}</span>
+                <button class="jobs-dropdown-arrow" mat-icon-button [matMenuTriggerFor]="menu" [class.visible]="showDropdownArrow(j)">
+                  <img src="{{ getDropdownArrowUrl() }}">
+                </button>
+              </div>
+              <mat-menu #menu="matMenu" class="wide-menu" xPosition="before" yPosition="below" [overlapTrigger]="false">
+                <mat-form-field>
+                  <input matInput
+                         #labelField
+                         value="{{ getFieldValue(j, df) }}"
+                         maxlength="255"
+                         (click)="$event.stopPropagation()">
+                  <mat-hint align="end">{{labelField.value?.length || 0}}/255</mat-hint>
```

Comment:
Where's the 255 coming from? Comment? Or pull from a constant and comment there?

Reply:
It could eventually go into capabilities, but that's probably overkill right now. I think a constant with clear documentation of the origin and current status of this limit is good for now.

Reply:
added – please let me know if it's not clear